langchain/utilities/scenexplain.py

```diff
@@ -23,7 +23,7 @@ class SceneXplainAPIWrapper(BaseModel):
       and create a new API key.
     """
 
-    scenex_api_key: str
+    scenex_api_key: str = Field(..., env="SCENEX_API_KEY")
```

Comment:
Let's it load from environment (useful if we're trying to load the tool using `load_tools()` for example)